apps/api/Dockerfile

```
@@ -0,0 +1,74 @@
+ENV PNPM_HOME="/pnpm"
+ENV PATH="$PNPM_HOME:$PATH"
+ENV CI=true
+
+RUN corepack enable
+
+# Build Go shared library
+FROM golang:1.24 AS go-build
+WORKDIR /app
+COPY sharedLibs/go-html-to-md ./sharedLibs/go-html-to-md
+
+RUN cd sharedLibs/go-html-to-md && \
```

Hadolint (2.14.0)

[warning] Use WORKDIR to switch to a directory

(DL3003)